src/azul/service/manifest_service.py

```diff
@@ -624,8 +641,39 @@ def __init_subclass__(cls) -> None:
             cls.cls_for_format[format] = cls
 
     @classmethod
-    def command_lines(cls, url: str) -> Optional[JSON]:
-        return None
+    def _cmd_exe_quote(cls, s: str) -> str:
```

Comment:
Do you have to move this method? If not, I'd prefer it to remain where it is.

Reply:
Isn't it necessary if we also want to apply its function to all command lines other than the `curl` format? Or I am not sure I understand your preference.

Reply:
Please give a definitive answer. I don't have the capacity to remember all the details.

Reply:
It's necessary in order to reduce duplication.

Reply:
Makes sense.